.gitignore

```diff
@@ -17,3 +17,5 @@ node_modules
 npm-debug.log
 
 coverage.html
+
+.idea
```

Comment:
This should probably not be part of our gitignore - it's not an artefact this project generates that we don't want to check in, its an editor arteface - and so should be in the user's `~/.gitignore`. 

Can we remove this line please?

Reply:
Of course.

Reply:
I have also remove the extraneous ... impements AssertionError<T> from the class declaration.